app/views/events/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: event, local: true) do |form| %>
+  <% if event.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</h2>
+
+      <ul>
+        <% event.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ] Create a helper method to put the if statement and create a partial to put the contents of the if statement, render the partial within the if statement in the helper method and call the method here instead of having the complexity straight in the view